### ISSUE TITLE
Disable autocomplete for input fields almost on every input field

### ISSUE THREAD

**Issue by raj454raj** [stopstalk/stopstalk-deployment#330]
Need to take a conscious where the user would want an browser default autocomplete functionality

**Comment by triden005** [comment]
Is this issue still open ?. Have to add autocomplete="false" to all input tags ?

**Comment by raj454raj** [comment]
Yes, where we need to add autocomplete="false" where we think auto complete is not helping the users

**Comment by triden005** [comment]
I am new to github pr's can you assign it to me?
